lab-dylan-sanders/README.md

```diff
@@ -0,0 +1 @@
+This repo creates a TCP Server that connects tracks multiple sockets in the cilentPool module
```

Comment:
Please expand more within your README:
I don't need a lot of words, I just need a short description of how to use you app.

For example:
Clone down then run in terminal:
```
npm i 
```
To run tests and lint files in terminal type:
```
gulp
```
To start server in terminal:
```
node server.js
```
To start a chat in terminal type:
```
telnet localhost 3000
```